selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/provider/OpenShiftWebConsoleUrlProvider.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.provider;
+
+import static java.lang.String.format;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.google.inject.Singleton;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.inject.Named;
+import org.eclipse.che.selenium.core.utils.UrlUtil;
+
+/** @author Dmytro Nochevnov */
+@Singleton
+public class OpenShiftWebConsoleUrlProvider implements Provider<URL> {
+
+  private static final int PORT = 8443;
```

Comment:
Is web console port always "8443" and can`t be reconfigured by any system variable?

Reply:
It can be reconfigured manually through the "OPENSHIFT_URL" environment variable.